### DISCUSSION TITLE
Show last modification times in dashboard and alert lists

### DISCUSSION THREAD

**Discussion by nagylzs** [grafana/grafana#62942] [General]
It would be nice to have a "last changed" column in alert and dashboard listings, and the ability to sort items by that column. Sometimes we need to go back and undo some changes from a given point of time, or just look at the list of things that have been changed on a certain day etc.

Examples: /alerting/list, /dashboards listing of dashboards in a given folder etc.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82909. Feel free to continue the discussion around this there. Thank you!
